lib/word_2_quiz/answer.rb

```diff
@@ -8,23 +10,24 @@ def initialize(text = "", correct = false)
     end
 
     def self.from_paragraphs(paragraphs, solution)
-      text = paragraphs.map(&:to_html).join("\n")
-      answer_letter = paragraphs[0].text.match(/^[a-z]/)
+      non_empty_paragraphs = Helpers.strip_blanks(paragraphs)
+      text = non_empty_paragraphs.map(&:to_html).join("\n")
+      answer_letter = non_empty_paragraphs[0].text.match(/^[a-z]/)
```

Comment:
Let's avoid magical numbers. If the answer letter will always position 0, then make a constant assigned to 0.
```ruby
ANSWER_LETTER = 0
non_empty_paragraphs[ANSWER_LETTER]
```